### PR TITLE
[New Rule] Execution from a Removable Media with Network Connection

### PR DESCRIPTION
**Pull request by Samirbous**
[This](https://github.com/elastic/protections-artifacts/blob/main/behavior/rules/initial_access_execution_from_a_removable_media_with_network_connection.toml) existing endpoint rule will be deprecated (due to unmanageable FP patterns), its still important for detection-only, thus replicating same logic to SIEM using Elastic Defend as an integration.

### CODE REVIEW

File: rules/windows/initial_access_execution_from_removable_media.toml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+maturity = "production"
+min_stack_comments = "Events Device metadata was added in Elastic Defend version 8.4 and above."
+min_stack_version = "8.4.0"
+updated_date = "2023/03/20"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies process execution from a removable media and by an unusual process. Adversaries may move onto systems, 
+possibly those on disconnected or air-gapped networks, by copying malware to removable media and taking advantage
```
Comment (w0rk3r) [inline rules/windows/initial_access_execution_from_removable_media.toml:13]:
```suggestion
possibly those with heavy network restrictions, by copying malware to removable media and taking advantage 
```

Comment (w0rk3r) [reply rules/windows/initial_access_execution_from_removable_media.toml:13]: Sounds like the network connection would not be possible in the scenario

**Review by w0rk3r — APPROVED** [review]
🧙🏼 🔥

**Review by imays11 — APPROVED** [review]

File: rules/windows/initial_access_execution_from_removable_media.toml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Execution from a Removable Media with Network Connection"
+risk_score = 47
+rule_id = "5dde2134-cf00-4716-9f82-f2203dd992e6"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Initial Access"]
```
Comment (w0rk3r) [inline rules/windows/initial_access_execution_from_removable_media.toml:24]:
```suggestion
tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Initial Access"]
```

File: rules/windows/initial_access_execution_from_removable_media.toml [hunk 3]
```diff
@@ -0,0 +1,66 @@
+[metadata]
+creation_date = "2023/03/20"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "Events Device metadata was added in Elastic Defend version 8.4 and above."
+min_stack_version = "8.4.0"
+updated_date = "2023/03/20"
```
Comment (w0rk3r) [inline rules/windows/initial_access_execution_from_removable_media.toml:7]:
```suggestion
updated_date = "2023/06/30"
```

**Review by w0rk3r — APPROVED** [review]

**Review by terrancedejesus — APPROVED** [review]

**Comment by w0rk3r** [comment]
Added as part of https://github.com/elastic/detection-rules/pull/3143

**Pull request closed without merging**